gateway/src/apicast/policy/rate_limit/apicast-policy.json

```diff
@@ -181,6 +181,7 @@
         "type": "array",
         "items": {
           "type": "object",
+          "required": ["key", "count", "window"],
```

Comment:
I see that in the other kind of limiters, the number used to limit is not required. I think it's the same case as `window` here. Have you checked them?

Reply:
Yup, tested the others and no division happens.